packages/venia-concept/src/components/ToastContainer/__stories__/toasts.js

```diff
@@ -32,7 +32,9 @@ const ToastEmitter = ({
     };
 
     useEffect(() => {
-        addToast(toastProps);
+        setTimeout(() => {
+            addToast(toastProps);
+        }, 0);
```

Comment:
Expectedly the logger doesn't make its way between every dispatch. If we queue a bunch of dispatches at once, like we do here in this story, we end up with a bunch of updates and only one state output. In dev this probably is OK cause most actions are one-offs.